package.py

```diff
@@ -354,7 +341,7 @@ def setup_miniupnpc(self) -> None:
         build_dir = self.__storage.build_directory
         os.chdir(build_dir)
         url = f'https://github.com/mrx23dot/miniupnp/releases/download/miniupnpd_2_2_24/{miniupnc}'
-        zip_filename = Downloader.download(url)
+        zip_filename, _ = urllib.request.urlretrieve(url, Path.joinpath(build_dir, miniupnc))
         zip_path = build_dir / zip_filename
```

Comment:
When setting the filename argument it is the path where the file will be saved and the returned value in this case is the same as the argument

```
>>> a, b = urllib.request.urlretrieve('https://google.com', '/home/yabirgb/google.html')
>>> a
'/home/yabirgb/google.html'
```

```suggestion
        zip_path = build_dir / miniupnc 
        urllib.request.urlretrieve(url, zip_path)
```